cassandra-diagnostics-core/src/main/java/io/smartcat/cassandra/diagnostics/api/HttpHandler.java

```diff
@@ -46,4 +65,8 @@ public Response serve(IHTTPSession session) {
         }
     }
 
+    private boolean hasValidCredentials(IHTTPSession session) {
+        String header = session.getHeaders().get("Authorization");
```

Comment:
According to the official docs / repo: "All header names are converted to lower case so they don't vary between browsers/clients."